src/Driver/DropzoneUploadDriver.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace LaraCrafts\ChunkUploader\Driver;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
+use LaraCrafts\ChunkUploader\Exception\UploadHttpException;
+use LaraCrafts\ChunkUploader\Helper\ChunkHelpers;
+use LaraCrafts\ChunkUploader\Identifier\Identifier;
+use LaraCrafts\ChunkUploader\Range\RequestRange;
+use LaraCrafts\ChunkUploader\Response\PercentageJsonResponse;
+use LaraCrafts\ChunkUploader\StorageConfig;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException;
+
+class DropzoneUploadDriver extends UploadDriver
+{
+    use ChunkHelpers;
+
+    /**
+     * @var string
+     */
+    private $fileParam;
+
+    public function __construct($config)
+    {
+        $this->fileParam = $config['param'];
+    }
+
+    /**
+     * @param \Illuminate\Http\Request $request
+     * @param Identifier $identifier
+     * @param StorageConfig $config
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     *
+     * @throws \Symfony\Component\HttpKernel\Exception\HttpException
+     */
+    public function handle(Request $request, Identifier $identifier, StorageConfig $config): Response
+    {
+        if ($this->isRequestMethodIn($request, [Request::METHOD_POST])) {
+            return $this->save($request, $identifier, $config);
+        }
+
+        throw new MethodNotAllowedHttpException([
+            Request::METHOD_POST,
+        ]);
+    }
+
+    /**
+     * @param \Illuminate\Http\Request $request
+     * @param \LaraCrafts\ChunkUploader\Identifier\Identifier $identifier
+     * @param StorageConfig $config
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function save(Request $request, Identifier $identifier, StorageConfig $config): Response
+    {
+        $file = $request->file($this->fileParam);
+
+        if (null === $file) {
```

Comment:
I'd use this, it makes the code more verbose:

```suggestion
        if (!$request->hasFile($this->fileParam)) {
```

Reply:
`hasFile` also checks if the file is valid. That means with this check the error statement is incorrect which is currently "File not found in request body".